nat-lab/docker-compose.yml

```diff
@@ -164,12 +165,11 @@ services:
     networks:
       internet:
         ipv4_address: 10.0.11.4
-      internet_v6:
-        ipv6_address: 2001:0db8:85a4::dead:beef:ceed
+        ipv6_address: 2001:db8:85a4::dead:beef:ceed
     environment:
       CLIENT_GATEWAY_PRIMARY: none
     extra_hosts:
-      - "photo-album:2001:0db8:85a4::adda:edde:0005"
+      - "photo-album:::adda:edde:0005"
```

Comment:
Change `:0005` to `:5` or change line 23 to match this.

Reply:
This seems like a typo, will fix.